ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -0,0 +1,203 @@
+                    std::shared_ptr<ngraph::Node> data_shape_node;
+                    if (data_pshape.is_static())
+                    {
+                        auto shape = data_pshape.to_shape();
+                        data_shape_node = default_opset::Constant::create(
+                            element::u64, Shape{shape.size()}, shape);
+                    }
+                    else
+                    {
+                        data_shape_node = std::make_shared<default_opset::ShapeOf>(data);
+                    }
+                    auto data_reshaped = std::make_shared<default_opset::Reshape>(
+                        data, detail::create_group_norm_shape(data, num_groups), true);
+                    const auto reduction_axes =
+                        common::get_monotonic_range_along_node_rank(data_reshaped, 2);
+                    auto mean = std::make_shared<default_opset::ReduceMean>(
+                        data_reshaped, reduction_axes, true);
+                    auto diff = std::make_shared<default_opset::Subtract>(data_reshaped, mean);
+                    auto variance = std::make_shared<default_opset::ReduceMean>(
+                        std::make_shared<default_opset::Power>(
+                            diff, default_opset::Constant::create(element::f32, Shape{}, {2})),
+                        reduction_axes,
+                        true);
+
+                    const std::shared_ptr<ngraph::Node> eps_node =
+                        std::make_shared<default_opset::Constant>(element::f32, Shape{}, eps);
+                    const auto sqrt = std::make_shared<default_opset::Sqrt>(
+                        std::make_shared<default_opset::Add>(variance, eps_node));
+
+                    auto data_rank_size = data.get_partial_shape().rank().get_length();
+                    auto scale_rank_size = scale.get_partial_shape().rank().get_length();
+                    if (scale_rank_size < data_rank_size)
+                    {
+                        scale = std::make_shared<default_opset::Reshape>(
+                            scale, detail::create_scale_bias_shape(scale, data_rank_size), true);
+                    }
+                    auto bias_rank_size = bias.get_partial_shape().rank().get_length();
+                    if (bias_rank_size < data_rank_size)
+                    {
+                        bias = std::make_shared<default_opset::Reshape>(
+                            bias, detail::create_scale_bias_shape(bias, data_rank_size), true);
+                    }
+
+                    std::shared_ptr<ngraph::Node> result =
+                        std::make_shared<default_opset::Divide>(diff, sqrt);
+                    result =
+                        std::make_shared<default_opset::Reshape>(result, data_shape_node, true);
+                    result = std::make_shared<default_opset::Multiply>(scale, result);
+                    result = std::make_shared<default_opset::Add>(result, bias);
```

Comment:
Could you check if this helper can be used here?
https://github.com/openvinotoolkit/openvino/blob/master/ngraph/frontend/onnx_import/src/op/conv.cpp#L80-L99